services/core-web/common/actionCreators/permitActionCreator.js

```diff
@@ -133,3 +133,38 @@ export const removePermitAmendmentDocument = (
     .catch(() => dispatch(error(reducerTypes.UPDATE_PERMIT_AMENDMENT_DOCUMENT)))
     .finally(() => dispatch(hideLoading()));
 };
+
+export const deletePermit = (mineGuid, permitGuid) => (dispatch) => {
+  dispatch(request(reducerTypes.DELETE_PERMIT));
+  return CustomAxios()
+    .delete(`${ENVIRONMENT.apiUrl}${API.PERMITDELETE(mineGuid, permitGuid)}`, createRequestHeader())
+    .then((response) => {
```

Comment:
update casing of 'PERMITDELETE` to be `PERMIT_DELETE` same goes for `PERMITAMENDMENT` below

Reply:
For consistency I followed the pattern of 'Permit' in API.js:
PERMITSTATUSCODES
PERMITAMENDMENTS
PERMITAMENDMENTDOCUMENT .....
Do we want to update all of them?

Reply:
yeah, they should all be snake_cased, They must have been overlooked.